### ISSUE TITLE
[JENKINS-71498] WCAG Operable:2.1 - Keyboard operation and navigation - Result 6. Orientation and navigation in the application are more difficult

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-23#7347]
<p>Orientation is made more difficult if content happens to have the same name (see image for example in breadcrumbs).</p>
<ul>
	<li>These problems make orientation and navigation considerably more difficult, especially for blind users who cannot use any visual orientation options, but who rely on the linear output of the screen reader, and represent an obstacle when using the application.</li>
	<li>The headings should be clearer and better structured, e.g. B. by always naming the level first and then the description of the content.</li>
	<li>The menus on the left should be better structured and designed more clearly.</li>
</ul>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cristinap">cristinap</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-71498" target="_blank">WCAG Operable:2.1 - Keyboard operation and navigation - Result 6. Orientation and navigation in the application are more difficult</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: accessibility
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-23
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Orientation is made more difficult if content happens to have the same name (see image for example in breadcrumbs).</p>
<ul>
	<li>These problems make orientation and navigation considerably more difficult, especially for blind users who cannot use any visual orientation options, but who rely on the linear output of the screen reader, and represent an obstacle when using the application.</li>
	<li>The headings should be clearer and better structured, e.g. B. by always naming the level first and then the description of the content.</li>
	<li>The menus on the left should be better structured and designed more clearly.</li>
</ul>
</pre>
</details>
</details>
<details><summary><i>1 attachment</i></summary>

- [Screen Shot 2023-06-20 at 2.39.19 PM.png](https://issues.jenkins.io/secure/attachment/60629/Screen%20Shot%202023-06-20%20at%202.39.19%20PM.png)
  > ![Screen Shot 2023-06-20 at 2.39.19 PM.png](https://issues.jenkins.io/secure/attachment/60629/Screen%20Shot%202023-06-20%20at%202.39.19%20PM.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `epic` from Jira: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-23/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-71153%22">JENKINS-71153</a>]</i>
